### DISCUSSION TITLE
Table of Contents

### DISCUSSION THREAD

**Discussion by realyukii** [facebook/docusaurus#9936] [Q&A]
Is there any example or plugin available to create a table of contents that resembles the one used in the Arch Linux wiki?

edited:
here's the TOCs of [arch wiki](https://wiki.archlinux.org/title/Tor#Chromium)
![03-10-2024](https://github.com/facebook/docusaurus/assets/126035476/a4b53e96-c9f2-4d66-be4c-a49508444dc8)

**Comment by slorber** [comment]
We don't provide any plugin apart the one generating the default TOC.

You can write your own remark plugin to generate a custom TOC if needed.

We don't know what the Arch Linux TOC look like, please post screenshots or at least a link or explanations of how their TOC works
